### DISCUSSION TITLE
Facerecognition Container

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#1182] [Ideas] [answered]
https://apps.nextcloud.com/apps/facerecognition

https://github.com/matiasdelellis/facerecognition/wiki/Docker

**Comment by Zoey2936** [comment]
I think this won't will be supported, see Point three: https://github.com/nextcloud/all-in-one#requirements-for-integrating-new-containers
But maybe I misunderstand this point, and they mean something else with this, so let's see. I'm not 100% sure.

**Comment by szaimen** [comment, marked as answer]
Yes, this will not be added. However I'm looking into this for the next release which allows to add dependencies into the container which is a requirement for being able to install this app. See https://github.com/nextcloud/all-in-one/issues/1162
